### PR TITLE
Revise how returned promises are described in readable streams

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

**Comment by jpmedley** [comment]
@reillyeon FYI

File: files/en-us/web/api/readablestreamdefaultcontroller/close/index.html [hunk 1]
```diff
@@ -1,109 +1,96 @@
 ---
-title: ReadableStreamDefaultController.close()
-slug: Web/API/ReadableStreamDefaultController/close
+title: ReadableStreamDefaultReader
```
Comment (reillyeon) [inline files/en-us/web/api/readablestreamdefaultcontroller/close/index.html:2]: What's going on in this file? It looks like the documentation for close() is being overwritten by the documentation for ReadableStreamDefaultReader.

**Comment by jpmedley** [comment]
@reillyeon I fixed the cut and paste error you noted. While I was doing that, I realized I should check how promises are done in all the stream interfaces. I made this PR about the readable streams.

File: files/en-us/web/api/readablestreamdefaultreader/cancel/index.html [hunk 2]
```diff
@@ -35,7 +33,7 @@ <h3 id="Parameters">Parameters</h3>
 
 <dl>
   <dt>reason {{optional_inline}}</dt>
-  <dd>A {{domxref("DOMString")}} providing a human-readable reason for the cancellation.
+  <dd>A {{domxref("DOMString")}} providing a human-readable reason for the cancellation. This value may or may not be used.
```
Comment (reillyeon) [inline files/en-us/web/api/readablestreamdefaultreader/cancel/index.html:36]: Same here.

**Review by reillyeon — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/readablestream/cancel/index.html [hunk 3]
```diff
@@ -2,19 +2,13 @@
 title: ReadableStream.cancel()
 slug: Web/API/ReadableStream/cancel
 tags:
-- API
-- Method
-- ReadableStream
-- Reference
-- Streams
-- cancel
+- API - Method - ReadableStream - Reference - Streams - cancel
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply files/en-us/web/api/readablestream/cancel/index.html:5]: I thought I indented. I don't know why that happened.

File: files/en-us/web/api/readablestream/cancel/index.html [hunk 4]
```diff
@@ -31,7 +25,7 @@ <h3 id="Parameters">Parameters</h3>
 
 <dl>
   <dt>reason</dt>
-  <dd>A {{domxref("DOMString")}} providing a human-readable reason for the cancellation.
+  <dd>A {{domxref("DOMString")}} providing a human-readable reason for the cancellation. The underlying source may or may not use it.
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply files/en-us/web/api/readablestream/cancel/index.html:28]: Fixed.

File: files/en-us/web/api/readablestreambyobreader/cancel/index.html [hunk 5]
```diff
@@ -32,7 +30,7 @@ <h3 id="Parameters">Parameters</h3>
 
 <dl>
   <dt>reason</dt>
-  <dd>A {{domxref("DOMString")}} providing a human-readable reason for the cancellation.
+  <dd>A {{domxref("DOMString")}} providing a human-readable reason for the cancellation. The underlying source may or may not use it.
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply files/en-us/web/api/readablestreambyobreader/cancel/index.html:33]: Done.

File: files/en-us/web/api/readablestreamdefaultreader/cancel/index.html (hunk 2 above)
Comment (jpmedley) [reply files/en-us/web/api/readablestreamdefaultreader/cancel/index.html:36]: Done.

**Comment by jpmedley** [comment]
@reillyeon PTAL

**Review by reillyeon — APPROVED** [review]

**Comment by jpmedley** [comment]
Google folks are done with this. Can we get a review from MDN at your convenience?

**Review by Elchi3 — APPROVED** [review]
LGTM, sorry for not seeing this earlier!

**Pull request merged**